dali/image/image.h

```diff
@@ -92,6 +92,15 @@ class Image {
     };
   }
 
+  inline void SetUseFastDct(bool use_fast_dct) {
```

Comment:
A general remark for this PR: Shouldn't it be "idct" everywhere? Or are we somehow preparing for JPEG _encoding_ and want to keep the flag name?

Reply:
Done